features/miniAlert.js

```diff
@@ -0,0 +1,25 @@
+//Thanks Vals

```

Comment:
there should be a space after the // and before the text in the comment

Reply:
you should add this into a class, add proper settings and add the file to the loader so it gets loaded

Reply:
Didn't Ask.